help/en/package/jmri/jmrit/withrottle/Protocol.shtml

```diff
@@ -265,11 +262,11 @@ PTL]\[LT12}|{Rico Station N}|{1]\[LT324}|{Rico Station S}|{2
       <h3 id="RouteList">Route List</h3>
 
       <p>Like for turnouts, JMRI will reply with zero, one or two strings. The first string is
-      likewise a list of labels for the route values. Again, Unknown (1) is not explicitly listed.
+      likewise a list of labels for the route values.
       For example:</p>
 
       <pre class="code">
-PRT]\[Routes}|{Route]\[Active}|{2]\[Inactive}|{4
+PRT]\[Routes}|{Route]\[Active}|{2]\[Inactive}|{4]\[Unknown}|{0]\[Inconsistent}|{8]\[Unknown}|{
```

Comment:
Should Unknown be added twice here?

Reply:
Yes, one has key of "0", one has key "" (empty string). In the WiThrottle code, the "" is sent when there is no sensor defined in the route, so no state can be determined. I chose not to change (fix?) that, as it might have unforeseen consequences. This PR is just putting labels on the stuff already being sent.

Reply:
UPDATE: Changed my mind and fixed the code to send the "1" and no longer send the "".